CHANGELOG.md

```diff
@@ -1,4 +1,106 @@
 # CHANGELOG
+## v0.2.0 | 2022-02-23
+The second minor release focuses on the stability and robustness of existing features via minor refactorings, mainly for
+headers synchronization and storing, while fixing the DA network segregation by providing bootstrap peers, allowing you to
+run a Light Node with zero effort. The release also includes a few breaking changes.
+
+### Highlights
+#### Hardcoded bootstrap peers
+Bootstrappers are essential backbone peers that every other peer is connected to be part of the DA p2p network.
```

Comment:
they don't necessarily *need* to be connected to a bootstrapper, it's just a default.

this might be confusing

Reply:
They have to be connected, otherwise they are disconnected from our network. What you wanted to say is probably that they don't have to use them as trusted peers to sync from - yes, but the connection to them should still present.

Reply:
We configure that auto connectivity here: https://github.com/celestiaorg/celestia-node/blob/main/node/p2p/routing.go#L43. This line ensures a DHT is connected to the network through the bootstrappers.

Reply:
Yeah I guess the language came off a bit confusing like you had to be connected to the default bootstrap peers to be connected to the network. Lemme make a suggestion